lib/types/application/ApplicationBuilder.js

```diff
@@ -171,7 +171,10 @@ class ApplicationBuilder extends AbstractBuilder {
 					workspace: resourceCollections.workspace,
 					dependencies: resourceCollections.dependencies,
 					options: {
-						namespace: project.metadata.namespace
+						namespace: project.metadata.namespace,
+						signatureType: project.builder
```

Comment:
@matz3 @RandomByte the configuration should be judged by you whether it fits to your concept

Reply:
LGTM

Reply:
Also fine from my side